.circleci/config.yml

```diff
@@ -101,6 +94,62 @@ jobs:
           docker build -t logindotgov/pki:$CIRCLE_TAG -f Dockerfile .
           echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
           docker push logindotgov/pki:$CIRCLE_TAG
+
+  check-expiring-certs-config:
+    docker:
+      # Specify the Ruby version you desire here
+      - image: circleci/ruby:2.6-node-browsers
+        environment:
+          RAILS_ENV: test
+          CC_TEST_REPORTER_ID: c88a6f4af1fbf80e0fc9a5593ebff124b2f940645b1eacb5adb681522bbf650e
+          COVERAGE: true
+          # PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      - image: circleci/postgres:9.5-alpine
+        environment:
+          POSTGRES_USER: circleci
+
+      - image: redis:4.0.1
+
+    working_directory: ~/identity-pki
+
+    steps:
+      - checkout
+      - install-deps
+      - run:
+          name: Test Setup
+          command: |
+            cp config/application.yml.example config/application.yml
+            bundle exec rake db:setup --trace
+      - run:
+          name: Check for expiring certs
+          command: |
+            bundle exec rake certs:print_expiring
+
+      - slack/status:
+          fail_only: true
+          failure_message: ":piv-card::red_circle::scream: identity-pki has certs expiring within 30 days"
+          include_project_field: false
```

Comment:
I don't have access to check, but am I correct in assuming we'll need to configure a webhook in the GitHub project settings for this?

Reply:
Yeah will have to add a config

Reply:
Just following up: I added a `SLACK_WEBHOOK` env var to this project